src/traces/violin/calc.js

```diff
@@ -42,13 +42,12 @@ module.exports = function calc(gd, trace) {
 
         // sample standard deviation
         var ssd = Lib.stdev(vals, len - 1, cdi.mean);
-        var bandwidthDflt = ruleOfThumbBandwidth(vals, ssd, cdi.q3 - cdi.q1);
-        var bandwidth = cdi.bandwidth = trace.bandwidth || bandwidthDflt;
+        var bandwidth = cdi.bandwidth = trace.bandwidth || ruleOfThumbBandwidth(vals, ssd, cdi.q3 - cdi.q1);
```

Comment:
So for your data I suspect `bandwidthDflt` equals 0. Is this correct?

Reply:
bandwidthDflt = 3.958842142868651e-8